templates/sysv/lsb-3.1/init.sh

```diff
@@ -88,7 +88,13 @@ stop() {
       sleep 1
     done
     if status ; then
-      emit "$name stop failed; still running."
+      if [ "$KILL_ON_STOP_TIMEOUT" = 1 ] ; then
```

Comment:
in bourne shell `[ x = y ]` is success (truthy) if the strings are equal. Not sure what bash == does.

I suppose technically we should use
```
if [ "$KILL_ON_STOP_TIMEOUT" -eq 1 ] ; then
```

since -eq forces a numeric comparison (so a value of 01, 001, etc should be truthful?)